### PR TITLE
Add CBL Mariner/Azure Linux `extra` images for .NET Runtime and ASP.NET Core

### LINKED ISSUE #5260: Add CBL Mariner/Azure Linux `extra` images for .NET Runtime and ASP.NET Core
These images would allow supporting globalization on CBL Mariner/Azure Linux distroless images without requiring AOT compilation or manual package installation. This also creates more symmetry between Ubuntu and CBL Mariner/Azure Linux images.

### Additional Context

- Needed for https://github.com/dotnet/dotnet-docker/issues/5258
- https://github.com/dotnet/dotnet-docker/issues/5021

### PR DESCRIPTION
**Pull request by lbussell**
Fixes #5260

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -4914,6 +4969,64 @@
             }
           ]
         },
+        {
+          "productVersion": "$(dotnet|9.0|product-version)",
+          "sharedTags": {
+            "$(dotnet|9.0|fixed-tag)-cbl-mariner2.0-distroless-extra": {},
+            "$(dotnet|9.0|floating-tag)-cbl-mariner2.0-distroless-extra": {},
+            "$(dotnet|9.0|floating-tag)-cbl-mariner-distroless-extra": {}
```
[the remark this answers is not in the document]
Comment (lbussell) [reply manifest.json:4977]: Should we remove these for other Mariner 9.0 images in a separate PR, and just exclude them for the new ones in this PR?

[the remark this answers is not in the document]
Comment (lbussell) [reply manifest.json]: Submitted #5268 for the image tags not in this PR.

File: src/aspnet/9.0/cbl-mariner2.0-distroless-extra/amd64/Dockerfile [hunk 2]
```diff
@@ -20,7 +20,7 @@ RUN aspnetcore_version=9.0.0-preview.2.24128.4 \
 
 
 # ASP.NET Core image
-FROM $REPO:9.0.0-preview.2-cbl-mariner2.0-distroless-extra-extra-amd64
```
Comment (mthalman) [inline src/aspnet/9.0/cbl-mariner2.0-distroless-extra/amd64/Dockerfile]: Read all about it. (Sorry, I couldn't resist)

**Review by mthalman — APPROVED** [review]

**Comment by MichaelSimons** [comment]
@lbussell - Can this issue be closed now?

**Pull request merged**